common/models/workspace.js

```diff
@@ -33,6 +33,15 @@ module.exports = function(Workspace) {
     var ViewDefinition = app.models.ViewDefinition;
     var TEMPLATE_DIR = path.join(__dirname, '..', '..', 'templates', 'projects');
     var DEFAULT_TEMPLATE = 'api-server';
+    var DEFAULT_LB_VERSION = '2.x';
+    var DEPENDENCY_3_X = {
+      'loopback': '^3.0.0-alpha.1',
+      'loopback-datasource-juggler': '^3.0.0-alpha.1',
```

Comment:
Ok I see, so could you provide more details about what is the entire `dependencies` in LB 3.x's `package.json`
Is it 
```json
'dependencies': {
    'compression': '^1.0.3',
    'cors': '^2.5.2',
    'helmet': '^1.3.0',
    'loopback': '^3.0.0-alpha.1',
    'loopback-boot': '^2.6.5',
    'loopback-component-explorer': '^2.4.0',
    'serve-favicon': '^2.0.1',
  },
```
?